web_poet/testing/pytest.py

```diff
@@ -0,0 +1,120 @@
+import asyncio
+import json
+import operator
+from pathlib import Path
+from typing import Any, Iterable, List, Optional, Set, Union
```

Comment:
coverage warnings are annoying, they basically tell that the module is never imported, which doesn't make sense